packages/galaxybudsclient-bin/galaxybudsclient-bin.pacscript

```diff
@@ -0,0 +1,38 @@
+name="galaxybudsclient-bin"
+gives="galaxybudsclient"
+pkgver="4.5.4"
+pkgdesc="Unofficial Galaxy Buds Manager for Windows and Linux"
+repology=("project: galaxybudsclient")
+homepage="https://github.com/ThePBone/GalaxyBudsClient"
+arch=("amd64" "arm64" "armhf")
+depends=("bluetooth")
+maintainer="Vigress <vig@disroot.org>"
```

Comment:
```suggestion
maintainer="vigress8 <vig@disroot.org>"
```
This makes it really easy for pacbot (check https://github.com/pacstall/pacstall-programs/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+author%3Aapp%2Fpacstall-pacbot for an idea of what it does) to ping you for outdated package reminders.